SConstruct

```diff
@@ -1429,8 +1429,7 @@ for libraryName, libraryDef in libraries.items() :
 	# osl shaders
 
 	def buildOSL( target, source, env ) :
-
-		subprocess.check_call( [ "oslc", "-I./shaders", "-o", str( target[0] ), str( source[0] ) ], env = env["ENV"] )
+		subprocess.check_call( [ env.subst( "$BUILD_DIR/bin/oslc" ), "-I./shaders", "-o", str( target[0] ), str( source[0] ) ], env = env["ENV"] )
```

Comment:
I suspect this could cause problems at Image Engine, where `oslc` is likely to be installed independently - @andrewkaufman? What's preventing Windows from using the PATH to find it?

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/4524/commits/404122426c090d1227df97d9042722c4e226483c